docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -6,94 +6,208 @@ description: "Efficiently allocate work through user tasks."
+
+#### Deploy and test run
+
+1. Click **Deploy** to deploy the process to your cluster.
    :::note
-   If you have not yet created a cluster, clicking **Deploy diagram** will take you to the console to create a cluster. Once you make your cluster creation request, you will automatically be redirected back to Modeler. The creation of a cluster can take 1 to 5 minutes. To read more about creating clusters, visit our documentation on [creating a cluster](create-cluster.md).
+   If you have not yet created a cluster, clicking **Deploy** will take you to Console to [create a cluster](create-cluster.md) first. Continue with this guide after cluster creation.
    :::
-2. To officially start a new process instance, click **Start instance**.
-3. Navigate to [Operate](/components/operate/operate-introduction.md) by clicking the honeycomb icon next to the **Start instance** button.
-4. Click **View process instances** to see your process instance alongside the green token waiting at the user task.
+2. After you deploy your process, it can be executed on the cluster. There are multiple ways to run a process. This time, click **Run** in Modeler for a test run.
 
-![token moving through process](./img/user-task-token-1.png)
+:::tip
+Other options to run a process are to start it via Tasklist, test it in the Play mode, or call it via the API or an inbound trigger. Read more about [run options](/components/modeler/web-modeler/run-or-publish-your-process.md).
+:::
 
-### Complete a user task
+#### Check successful start in Operate
 
-Within this example, we've included a form to demonstrate the completion of a human task. To learn more about creating forms within your diagrams, visit our guide on [building forms with Modeler](/components/modeler/bpmn/user-tasks/user-tasks.md#user-task-forms).
+1. The process start will be confirmed via a notification message on the screen. Click the **chevron icon** next to **Run** to open more options. Click **View process instances** to see the running process in Operate.
+   <img src={RunProcessImg} style={{width: 300}} alt="Run action in Modeler" />
 
-1. Go back to your Camunda Platform 8 diagram and select the honeycomb icon and then **View user tasks** to take a look at your user tasks inside Tasklist.
-2. Select the open user task on the left panel of **Tasks**. In our example below, this is **Decide what's for dinner**.
-3. Next to **Assignee**, click **Claim** to claim the task.
-4. Once finished entering the appropriate information, click **Complete Task**.
-   ![complete a human task in Tasklist](./img/user-task-tasklist.png)
-5. On the left panel of **Tasks**, filter by **Completed** tasks to see your task has been finished.
+2. In Operate, you will see a visualization of the running process instance. Notice that a green **token** is waiting at the user task. This means that a task is waiting to be worked on in Tasklist.
```

Comment:
Would link to Operate documentation here, and perhaps note how a user navigates to Operate. If they leave Modeler, how can they get back to their model? Should they open Operate in a separate tab? Is my work saved? I know the answers to these questions, but a user may want details before migrating across apps.

Reply:
There are some initiatives by the team that manages the navigation bar to change the behavior, and I don't know when this will happen. I would rather not document it here since we would have to monitor the change and update docs again

Reply:
But I added the following admonition to an earlier section:

:::note
Don't worry to save your process diagram—Modeler automatically saves every change you make.
:::